### ISSUE TITLE
Can't Post To News Channels

### ISSUE THREAD

**Issue by Corard** [SirWaddles/JohnWick#5]
A few weeks ago we got news and store channels on our server, and the bot was working just fine, now it doesnt seem to work, not even responding to commands in that channel

**Comment by SirWaddles** [comment]
I'm not really sure what would cause this, is there anything different about those channels? There must be something different about them.

The only thing I can think of is to absolutely make sure that the bot has permission to post in those channels, it should be responding to `!help` and `!subscribe` whenever the bot is in the channel, these are pretty simple functions.

**Comment by Corard** [comment]
Yeah I've double checked the permissions on the channel, making our bot role and JohnWick itself have permission to read and send to those channels. Our music bot responds to !help however JohnWick doesn't, yet in other channels it's able to

**Comment by SirWaddles** [comment]
I have no idea then. There would have to be something funky going on with those channels in particular. I mean, this is the code for the help message:
```javascript
    if (parts[0] == '!help') {
        msg.channel.send(HELP_MESSAGE);
        return;
    }
```

**Comment by Corard** [comment]
would you like to join the server and perhaps take a look?

**Comment by Corard** [comment]
standard webhooks seem to send perfectly fine

**Comment by Corard** [comment]
Closed as issue is no longer relevant
